static/templates/admin_user_group_list.handlebars

```diff
@@ -0,0 +1,24 @@
+{{#with user_group}}
+<div class="user-group" id="{{id}}">
+    <h4>
+        <span class="name" data-placeholder="{{t 'Name' }}" contenteditable="true" spellcheck="false">{{name}}</span>
+        —
+        <span class="description" data-placeholder="{{t 'Description' }}" contenteditable="true">{{description}}</span>
+        <button class="button rounded small save-group-changes sea-green">
+            <i class="fa fa-check" aria-hidden="true"></i>
+        </button>
+        <button class="button rounded small delete btn-danger">
```

Comment:
@adnrs96 I think you're looking at an outdated commit? The `icon-vector-trash` is already `fa-trash-o`